modules/ting_search/ting_search.module

```diff
@@ -398,6 +400,7 @@ function ting_search_search_execute($keys = NULL, $conditions = NULL) {
 
       $query = $query->withPage($page + 1)->withCount($results_per_page);
       $search_result = $query->execute();
+      $search_result->conditions = $conditions;
```

Comment:
This bolts an property to an object which is not defined for the class. This is a practice which should be avoided. 

As I understand it we do this to be able to determine which facets were set and with what value. This allows the facet browser to determine if a year facet has been selected. However this data should already be available in `$search_result->getSearchRequest()->getFilters()`.

I see a couple of options for how to deal with this:

1. Pass the filter objects instead of the conditions in `ting_search_ding_facetbrowser`.
2. Change `ding_facetbrowser` implementations to pass the entire `TingSearchResultInterface` object. This already have facets, search key and filters available.
3. Extract the conditions from the URL instead of from arguments in `ding_facetbrowser_form()`. This would follow the current pattern in the current implementation as seen in `_ding_facetbrowser_is_facet_set()`. It is not very elegant though.

If you go with 1 or 2 there is bonus points for also addressing the current implementation in 3 😅 .